modules/metering-uninstall-crds.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-uninstall.adoc
+
+[id="metering-uninstall-crds_{context}"]
+= Uninstalling metering CustomResourceDefinitions
+
+The metering CustomResourceDefinitions (CRDs) remain in the cluster after the Metering
+Operator is uninstalled and the `openshift-metering` namespace is deleted.
+
+[IMPORTANT]
+====
+Deleting the metering CRDs disrupts any additional metering installations in other namespaces.
+Ensure that there are no other metering installations before proceeding.
+====
+
+.Prerequisites
+
+*  The MeteringConfig custom resource in the `openshift-metering` namespace is deleted.
+*  The `openshift-metering` namespace is deleted.
+
+.Procedure
+
+*  Delete the remaining metering CRDs:
++
+[source,terminal]
+----
+$ oc get crd -o name | grep "metering.openshift.io" | xargs oc delete
```

Comment:
@pruan-rht did you ever check if OLM deletes the labels from metering CRDs when you uninstall metering from OLM? If not, we could use a label selector so everything can be done using only the `oc` binary.

Reply:
@timflannagan1 I delete the CRDs first by label before I remove the operator

Reply:
> @timflannagan1 I delete the CRDs first by label before I remove the operator

@timflannagan1 @pruan-rht Do this impact the command needed to delete the CRDs? Also, if the CRDs are deleted first by label, should this CRDs section move up in the _Uninstalling metering_ assembly to be one of the first steps? Thanks!

Reply:
@pruan-rht I tried locally on the shared cluster spinning up a metering olm installation and it doesn't look like OLM creates those labels if those CRDs already exist?

```bash
$ oc get crds --show-labels | grep metering
hivetables.metering.openshift.io                                  2020-09-03T17:08:33Z   <none>
meteringconfigs.metering.openshift.io                             2020-09-03T17:08:33Z   <none>
prestotables.metering.openshift.io                                2020-09-03T17:08:33Z   <none>
reportdatasources.metering.openshift.io                           2020-09-03T17:08:33Z   <none>
reportqueries.metering.openshift.io                               2020-09-03T17:08:33Z   <none>
reports.metering.openshift.io                                     2020-09-03T17:08:33Z   <none>
storagelocations.metering.openshift.io                            2020-09-03T17:08:33Z   <none>
```

Maybe we need to try on a cluster that isn't going to be contending with other metering installations, but I'm not sure if we should be relying on populated labels in this place.

Reply:
@timflannagan1 @pruan-rht Bumping to see if you all have any new clarification on this.

Reply:
@lbarbeevargas there's no guarantee those labels are going to exist. Let's use the command that we currently have documented that pipes output to xargs to delete.